cropharvest/inference.py

```diff
@@ -0,0 +1,101 @@
+from cropharvest.engineer import Engineer
+from cropharvest.utils import TORCH_INSTALLED
+from datetime import datetime
+from pathlib import Path
+from typing import Dict, List, Optional, Union
+import numpy as np
+import xarray as xr
+import pandas as pd
+import re
+
+if TORCH_INSTALLED:
+    import torch
+
+
+class Inference:
+    """
+    Class for running inference on a single tif file using an sklearn or
+    pytorch model (including jit).
+    """
+
+    def __init__(
+        self,
+        model,
+        device=None,
+        batch_size: int = 64,
+        normalizing_dict: Optional[Dict[str, np.ndarray]] = None,
```

Comment:
Maybe the normalizing dict can not have a default argument?

i.e. `normalizing_dict: Optional[Dict[str, np.ndarray]]` but without the `=  None`. 

This way it can be `None` but that has to be explicitly passed (instead of being an argument which can be ignored)

Reply:
Good call